vms/vm-under-test/scripts/customize-vm

```diff
@@ -90,21 +90,24 @@ cat <<'EOF' > "$checkup_boot_script_full_path"
 # Copyright 2024 Red Hat, Inc.
 #
 
-set -e
+set -x
```

Comment:
Could you please state in the commit message what is the reason for this change?

Reply:
DONE

Reply:
The reason why `set -e` was removed is explained, but the reason why `set -x` is required is missing.

Reply:
Right, DONE